frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -23,23 +22,47 @@ import { PodModel } from '@console/internal/models';
 
 import { VMKind, VMIKind } from '../../types';
 import { VMTemplateLink } from '../vm-templates/vm-template-link';
+import { getId, prefixedId } from '../../utils';
+import { vmDescriptioneModal } from '../modals/vm-description-modal';
+import { getDescription } from '../../selectors/selectors';
 
-export const VMResourceSummary = ({ vm }: VMResourceSummaryProps) => {
+import './_vm-resource.scss';
+
+export const VMResourceSummary = ({ vm, canUpdateVM }: VMResourceSummaryProps) => {
   const template = getVmTemplate(vm);
 
+  const id = getId(vm);
+  const description = getDescription(vm) || DASH;
+
   return (
     <ResourceSummary resource={vm}>
-      <dt>Description</dt>
-      <dd>{getDescription(vm)}</dd>
+      <dt>
+        Description
+        {canUpdateVM && (
+          <button
+            type="button"
+            className="btn btn-link co-modal-btn-link co-modal-btn-link--left"
+            onClick={() => vmDescriptioneModal({ vmLikeEntity: vm })}
+          />
+        )}
+      </dt>
+      <dd id={prefixedId(id, 'description')} className="kubevirt-vm-resource-summary__description">
+        {description}
+      </dd>
       <dt>Operating System</dt>
-      <dd>{getOperatingSystemName(vm) || getOperatingSystem(vm) || DASH}</dd>
+      <dd id={prefixedId(id, 'os')}>
+        {getOperatingSystemName(vm) || getOperatingSystem(vm) || DASH}
+      </dd>
       <dt>Template</dt>
-      <dd>{template ? <VMTemplateLink template={template} /> : DASH}</dd>
+      <dd id={prefixedId(id, 'template')}>
+        {template ? <VMTemplateLink template={template} /> : DASH}
+      </dd>
     </ResourceSummary>
   );
 };
 
 export const VMDetailsList = ({ vm, vmi, pods, migrations }: VMResourceListProps) => {
```

Comment:
can you fix this one too ? 
```suggestion
export const VMDetailsList: React.FC<VMResourceListProps> = ({ vm, vmi, pods, migrations }) => {
```